src/Surfnet/StepupMiddleware/MiddlewareBundle/Resources/config/console_commands.yml

```diff
@@ -0,0 +1,17 @@
+services:
+    Surfnet\StepupMiddleware\MiddlewareBundle\Console\Command\AbstractBootstrapCommand:
```

Comment:
Please don't use an abstract constructor

Reply:
For now I lean towards leaving it as is. I know its not common practice to use a shared constructor. But given the fact they actually all use the same dependencies. This is something I can get away with (imho). Once dependencies start to differ, this should definitely be reverted.